appengine/deploy/StudyManager.yaml

```diff
@@ -0,0 +1,15 @@
+service: study-manager-backend
+runtime: java11
+instance_class: B4
+
+manual_scaling:
+  instances: 1
```

Comment:
If we're doing manual scaling, we should definitely implement GAE's lifecycle call `/_ah/start`, otherwise GAE will continuously reboot the instance. Since `DSMServer` is running a Spark server, it should be returning `404` right now, which GAE's documentation says is fine. But I'll recommend adding a route and returning a `200`.

Reply:
Implementing /_ah/start is a good idea. Don't think it is as important for manual scaling as there will be no up/downscaling, but it is not hard to do. Check Yufengs implementation; wait to respond to /_ah/start request until app is completely initialized.
/_ah/stop too if there is cleanup needed when shutting down.